src/tvheadend.h

```diff
@@ -561,6 +561,8 @@ typedef enum {
 #define SM_CODE_NO_INPUT                  402
 #define SM_CODE_NO_SPACE                  403
 
+#define SM_CODE_PREVIOUSLY_RECORDED       500
```

Comment:
I'm not sure this code is being used since it seems you use SM_CODE_OK?

But I _think_ using this might require changes to htsp (for Kodi frontend) to map it for older clients (who don't know what a code 500 is). It _might_ be worth using a code in the 300 range such as 301 (since 300 is ABORTED and this is a type of abort).

Reply:
This is a mistake, I had tried to have a self code, but I would have to do more changes, so I prefered to send as SM_CODE_OK, but I forgot to remove it. I'm sorry.